core/src/main/scala/cats/Selective.scala

```diff
@@ -1,9 +1,10 @@
 package cats
 
+import cats.syntax.applicative.catsSyntaxApplicativeId
```

Comment:
This is so you can write things like `true.pure[F]`, right? I'd rather avoid this and stick to `pure(true)`.